contributing.md

```diff
@@ -306,6 +306,9 @@ ansible-playbook --skip-tags tests::cleanup \
     -i testhost, tests/playbooks/tests_802_1x.yml
 ```
 
+### NetworkManager Documentation
+- [NM 1.0 (1.27.91)](https://lazka.github.io/pgi-docs/#NM-1.0)
```

Comment:
The version will change, so it seems to be better not to include it there. Also, it would be nice to add a sentence that explains what the developer can read there.

Reply:
updated